### PR TITLE
Fixes #7650: Document AIX support in Rudder manual

### PR DESCRIPTION
**Pull request by amousset**
http://www.rudder-project.org/redmine/issues/7650

### CODE REVIEW

File: 00_introduction/21_architecture_nodes.txt [hunk 1]
```diff
@@ -33,18 +33,24 @@ available for these platforms:
...
 
 [[install-on-windows, Install on Microsoft Windows]] 
 
-.Windows Nodes 
+.Windows and AIX Nodes
 
 ====
 
-Installing Rudder on Windows requires the commercial version of CFEngine (named Nova).
+Installing Rudder on Windows or AIX requires a commercial version the Rudder agent.
```
Comment (amousset) [inline 00_introduction/21_architecture_nodes.txt]: Should we explain the difference between the Nova agent for Windows and AIX support?

Comment (VinceMacBuche) [inline 00_introduction/21_architecture_nodes.txt]: I think it can be explained, not in details but the reasons are different :)

**Comment by amousset** [comment]
PR updated.

File: 00_introduction/21_architecture_nodes.txt [hunk 2]
```diff
@@ -33,18 +33,26 @@ available for these platforms:
...
 [[install-on-windows, Install on Microsoft Windows]] 
 
-.Windows Nodes 
+.Windows and AIX Nodes
 
 ====
 
-Installing Rudder on Windows requires the commercial version of CFEngine (named Nova).
+* On Windows, installing Rudder requires the commercial version of CFEngine (named Nova)
+* On IBM AIX, installing Rudder requires a commercial version of the Rudder agent
```
Comment (jooooooon) [inline 00_introduction/21_architecture_nodes.txt]: Well. This is not strictly true, the AIX agent is open source. Normation just does not provide built packages of it.

Could you rephrase this to explain that to get a RPM package for AIX you need to contact Normation to obtain a demo version etc?

**Comment by amousset** [comment]
PR updated

File: 00_introduction/21_architecture_nodes.txt [hunk 3]
```diff
@@ -33,18 +33,27 @@ available for these platforms:
...
 
-.Windows Nodes 
+.Windows and AIX Nodes
 
 ====
 
-Installing Rudder on Windows requires the commercial version of CFEngine (named Nova).
+* On Windows, installing Rudder requires the commercial version of CFEngine (named Nova)
+* For IBM AIX, the agent is open-source but the pre-built RPM packages are distributed
+  by Normation under commercial license.
```
Comment (jooooooon) [inline 00_introduction/21_architecture_nodes.txt]: Hmmm. Almost :)

Actually we don't change the license when distributing (just like Red Hat "sells" DVDs of open source software). Could you rephrase to clarify as in "For IBM AIX, pre-built RPM packages are distributed by Normation only."

And while you're here, can you s/Nova/Enterprise/ ? :)

**Comment by amousset** [comment]
PR updated
